pulsar-io/netty/pom.xml

```diff
@@ -68,13 +68,6 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
-
-        <dependency>
```

Comment:
I notice that the pulsar-io-netty requires the TestNG. Why remove this dependency?

Reply:
Dependency is duplicated in file(s): Pulsar、Pulsar-io/netty、structured-event-log. And there are already duplicate dependencies in the root file pom.xml, open source considers using maven dependency delivery to solve the problem of duplicate dependencies

Reply:
@nodece test scope TestNG is globally depended in the root `pom.xml`.